module_prototyper/templates/8.0/models/model_name.py.template

```diff
@@ -0,0 +1,68 @@
+{% extends "header.template" %}
+{% block body %}
+
+from openerp import models, fields
+from openerp.tools.translate import _
```

Comment:
It's idiomatic of v8 to have all the usual imports on the same line from openerp.
Personally, I find it much nicer to read.